### DISCUSSION TITLE
How multiple DNS Blocklists are evaluated?

### DISCUSSION THREAD

**Discussion by logicito** [AdguardTeam/AdGuardHome#7167] [General]
I have 4 DNS Blocklists configured, how are they evaluated, AdGuard DNS filter first, then HaGeZi Threat, etc. or all of them are combined and then performed a single scan? Thanks
My lists are:
- AdGuard DNS filter
- HaGeZi's Threat Intelligence Feeds
- HaGeZi's Pro++ Blocklist
- OISD Blocklist Big

**Comment by Dynamic5912** [comment]
> I have 4 DNS Blocklists configured, how are they evaluated, AdGuard DNS filter first, then HaGeZi Threat, etc. or all of them are combined and then performed a single scan? Thanks
> 
> My lists are:
> 
> - AdGuard DNS filter
> 
> - HaGeZi's Threat Intelligence Feeds
> 
> - HaGeZi's Pro++ Blocklist
> 
> - OISD Blocklist Big

Regardless - if you're using Hagezi Pro++ you do not need OISD or AdGuard lists as well since they are already incorporated into Hagezi's lists.

**Reply by logicito** [reply]
@Dynamic5912 I have since move to **HaGeZi Pro**, instead of HaGeZi Pro++, that means I don't meed AdGuard DNS? I just keep HaGeZi Pro? Thank you

**Reply by Dynamic5912** [reply]
> @Dynamic5912 I have since move to **HaGeZi Pro**, instead of HaGeZi Pro++, that means I don't meed AdGuard DNS? I just keep HaGeZi Pro? Thank you

If you're using Hagezi's lists you do not need others from different list providers.
